test/box/misc.test.lua

```diff
@@ -348,3 +348,19 @@ tuple_format = box.internal.new_tuple_format(format)
 box.cfg{}
 local ffi = require'ffi' ffi.C._say(ffi.C.S_WARN, nil, 0, nil, "%s", "test log")
 test_run:grep_log('default', 'test log')
+
+--
+-- gh-2866: one more way to declare index parts
+--
+s = box.schema.space.create('test')
+i = s:create_index('test1', {parts = {{1, 'unsigned'}}})
+i = s:create_index('test2', {parts = {{2, 'string', is_nullable = true, collation = 'unicode'}}})
+i.parts[1].is_nullable
```

Comment:
As for me, instead of several lines `i = <...> \n i.parts[1].is_nullable \n ... ` it would be better to make one short line `<...>.parts`. It will simply show how part options was parsed.

Reply:
Fixed